packages/jahia/perf-test-step1.yml

```
@@ -11,9 +11,21 @@ globals:
 
 mixins:
   - ../../mixins/common.yml
+  - ../../mixins/jahia.yml
 
 onInstall:
-  - if("${nodes.proc.first.version}" >= "8.0.0.0"):
+  - getJahiaVersion
+  - isVersionHigher:
+      a: ${globals.jahiaVersion}
+      b: 8.0.0.0
+      res: isVersionHigherThan8000
+
+  - isVersionEqual:
+      a: ${globals.jahiaVersion}
+      b: 8.0.0.0
+      res: isVersionEqualTo8000
+
+  - if( ${globals.isVersionHigherThan8000} || ${globals.isVersionEqualTo8000}):
```

maybe we can do something like that here (or create `isVersionHigherOrEqual` and `isVersionLowerOrEqual`, sure they will be useful)

```suggestion
  - Compare:
      a: ${globals.jahiaVersion}
      b: 8.0.0.0
      res: isVersionHigherOrEqual8000

  - if(globals.isVersionHigherOrEqual8000 >= 0):
```

---

Personally I think that creating `isVersionHigherOrEqual` and `isVersionLowerOrEqual` (and also renaming the others 2 methods into `isVersionStrictlyHigher` and `isVersionStriclyLower`) would help make things easier to understand ; btw that's what we did on the frontend side. When I started my review I had to check the action definition to see if the condition was strict or not...

An alternative in your case @amitc-jahia would have been to check this instead:
```
  - isVersionLower:
      a: ${globals.jahiaVersion}
      b: 8.0.0.0
      res: isVersionLowerThan8000
  - if( ! ${globals.isVersionLowerThan8000} ):
```
But this isn't the easiest to understand, having multiple actions with explicit names is the way to go IMHO.

---

Fixed